doc/userguide/performance/ignoring-traffic.rst

```diff
@@ -66,10 +64,24 @@ Example:
   suppress gen_id 0, sig_id 0, track by_src, ip 1.2.3.4
 
 
-Encrypted traffic
+encrypted traffic
 -----------------
 
 The TLS app layer parser has the ability to stop processing encrypted traffic
 after the initial handshake. By setting the `app-layer.protocols.tls.encryption-handling`
 option to `bypass` the rest of this flow is ignored. If flow bypass is enabled,
 the bypass is done in the kernel or in hardware.
+
+bypassing traffic
+-----------------
+
+There are currently three ways to bypass traffic in addition to using the ``bypass`` rule keyword.
```

Comment:
I think this listing mixes to things:

1. what triggers a bypass (e.g. depth and encrypted traffic depending on config)
2. where is the bypass handled. This can be internally in suricata (in the flow engine) or in kernel, or in driver/nic with XDP and eBPF